client/src/main.tsx

```
@@ -1,15 +1,13 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client'
-import { BrowserRouter } from "react-router-dom";
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
-import './index.css'
-
-
+import './index.css';
 
 ReactDOM.createRoot(document.getElementById('root')!).render(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)